app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt

```diff
@@ -0,0 +1,84 @@
+package com.hedvig.app.feature.home.ui.claimstatus.composables
+
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.datasource.CollectionPreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.feature.home.ui.claimstatus.data.ClaimStatusColors
+import com.hedvig.app.feature.home.ui.claimstatus.data.PillData
+import com.hedvig.app.ui.compose.composables.pill.OutlinedPill
+import com.hedvig.app.ui.compose.composables.pill.Pill
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import com.hedvig.app.util.compose.preview.previewData
+
+@Composable
+fun ClaimPills(
+    pillData: List<PillData>,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.Top,
+    ) {
+        Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+            pillData.forEach { pillData: PillData ->
+                ClaimPill(
+                    text = pillData.text,
+                    pillType = pillData.type,
+                )
+            }
+        }
+        // TODO: Uncomment and check figma layout when the card is clickable to go to detail screen
+//        Icon(Icons.Default.ArrowForward, contentDescription = null)
+    }
+}
+
+@Composable
+private fun ClaimPill(
+    text: String,
+    pillType: PillData.PillType,
+) {
+    when (pillType) {
+        PillData.PillType.OPEN -> OutlinedPill(text)
+        PillData.PillType.CLOSED -> Pill(text, MaterialTheme.colors.primary)
+        PillData.PillType.REOPENED -> Pill(text, ClaimStatusColors.Pill.reopened)
+        PillData.PillType.PAYMENT -> Pill(text, ClaimStatusColors.Pill.paid)
+        PillData.PillType.UNKNOWN -> OutlinedPill(text)
+    }
+}
+
+@Preview
+@Preview(uiMode = UI_MODE_NIGHT_YES)
```

Comment:
Does this create two `showkase`-previews? 🤔 Also, should it be grouped?

Reply:
This creates *no* showkase previews actually https://github.com/airbnb/Showkase/issues/171.
Which is precisely why https://github.com/HedvigInsurance/android/pull/808/files#diff-d010a954ceb4addb5e1128f4097feeead5a18bc07e2f65ddfe08e41eedc69f8eR62-R79 exists which puts the entire card in showkase with the appropriate grouping etc.
Is this fine? Any other suggestions/proposals if not?

Reply:
Yes, it's all good